pkg/kinflate/commands/inflate.go

```diff
@@ -83,20 +86,35 @@ func (o *inflateOptions) Complete(cmd *cobra.Command, args []string) error {
 
 // runInflate does the real transformation.
 func (o *inflateOptions) runInflate(fs fs.FileSystem) (types.KObject, error) {
-	// Build a tree of ManifestData.
-	loader := tree.ManifestLoader{FS: fs, InitialPath: o.manifestPath}
-	root, err := loader.LoadManifestDataFromPath()
+	l := loader.Init([]loader.SchemeLoader{loader.NewFileLoader(fs)})
+
+	absPath, err := filepath.Abs(o.manifestPath)
+	if err != nil {
+		return nil, err
+	}
+
+	rootLoader, err := l.New(absPath)
 	if err != nil {
 		return nil, err
 	}
 
-	// Do the transformation for the tree.
-	err = root.Inflate(o.mode)
+	application, err := app.New(rootLoader)
+	if err != nil {
+		return nil, err
+	}
+
+	allResources := types.ResourceCollection{}
+	switch o.mode {
+	case tree.ModeNormal:
```

Comment:
Yeah, I think we should handle diff differently, but sure for now.

Reply:
I have another change for removing `o.mode`. Then `diff` and `inflate` will be handled separately.